### PR TITLE
Migrate shared folders to virtual hosts

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#5234

### CODE REVIEW

**Review by filippocarletti — COMMENTED** [review]
Should /var/www be added to backup-data?

**Comment by gsanchietti** [comment]
No we shouldn't because that directory can contain data belonging to some rpms (eg. owncloud).
We need to craft a a special template to handle such backup scenario.

**Review by gsanchietti — COMMENTED** [review]
Reset ACL on migration action.

File: root/usr/share/nethesis/NethServer/Module/VirtualHosts/ModifyDefault.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ */
+class ModifyDefault extends \Nethgui\Controller\Table\RowAbstractAction
+{
+
+    public function initialize()
+    {
+
+        $schema = array(
+            array('name', Validate::USERNAME, Table::KEY),
+#            array('Description', Validate::ANYTHING, Table::FIELD),
```
Comment (gsanchietti) [inline root/usr/share/nethesis/NethServer/Module/VirtualHosts/ModifyDefault.php]: Remove this line

**Pull request merged**